<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_export</class>
 <widget class="QDialog" name="Data_export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eksport danych</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 200, 200);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_box">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="left_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(223, 223, 223)</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_copy_to_clipboard">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(223, 223, 223)</string>
       </property>
       <property name="text">
        <string>Kopiuj</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="right_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_ok</sender>
   <signal>clicked()</signal>
   <receiver>Data_export</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
